### ISSUE TITLE
Missing GUI Elements - Latest Version (V2 and 

### ISSUE THREAD

**Issue by SunflowerTwix** [bmaltais/kohya_ss#1256]
Normally in the following location you would see two checkboxes:

[] v2      [] v_parameterization 

![image](https://github.com/bmaltais/kohya_ss/assets/13431506/d3927c2c-1dc9-4d4d-8278-427e200e6422)

As you can see, in this screenshot they are missing and seem to be replaced by an empty 'textbox' GUI element.
I have reinstalled Kohya_ss, installed the previous version and ensured Python is on the latest recommended version, reinstalled the latest Kohya_ss again on top of that (to rule out Python entirely).
Note: I have no issues with the GUI on A1111.

- Tried installing as far back as 21.7.5, and the checkboxes appear, however, the GUI doesn't appear to function. Clicking does nothing Clicking 'Train model' doesn't do anything in console, nothing changes in task manager to indicate activity change.

I'm happy to admit I'm somehow the issue here. Nobody else seems to have reported this issue, so I'm going to assume that I may be the exception.

Any ideas on where to start with this?

Thanks so much. I've had nothing but issues with trying to create my own LoRA since I started messing around with AI generated art. Every video I've watched, I've meticulously followed to the dot...but still, missing checkboxes, or errors (wrong adamw), or because no checkbox, resolution errors in use of the trained LoRA.

Thanks!

**Comment by SunflowerTwix** [comment]
Other simple troubleshooting I've done, none of which change the result of no checkboxes:
- Restarted
- Changed window sizes & scaling (thinking it might've been an incorrectly scaled GUI element)
- Launched GUI from different monitor
- Checked F12 elements and compared to older Kohya version. Older version had checkboxes in HTML elements, but not the newer version
- No errors on console as GUI starts:

![image](https://github.com/bmaltais/kohya_ss/assets/13431506/3dadf73b-b056-4b3e-a5b0-47d0cd7eeadc)

I don't see why external modules would interfere. I'll check to ensure I don't have some form of outdated Tkinter installed and report if any changes of removing it. (if so it would be from former python development)

**Comment by Signorlimone** [comment]
try this, in model quickpick select "custom"

**Comment by SunflowerTwix** [comment]
> try this, in model quickpick select "custom"

Hey, thanks for responding.
Unfortunately it didn't change anything. :(

**Comment by bmaltais** [comment]
Changing the model quick pick to custom should bring those options up... I will need to test it out when I get back home in a few days... But I am pretty sure this work fine.

**Comment by SunflowerTwix** [comment]
> Changing the model quick pick to custom should bring those options up... I will need to test it out when I get back home in a few days... But I am pretty sure this work fine.

Ah yes. They are appearing now. Thank you.

I looked after @Signorlimone's suggestion but it wasn't appearing (I closed the instance after that). But out of persistence I opened it back up again and voila. There it is. Thank you!

I'll give it a go and see if I get any errors (such as the resolution errors indicating V2 not being ticked), but I'm sure it'll be fine. Thanks again so much!

If I may provide that with some teeny tiny feedback, it is a little odd to need to select Custom from the dropdown. When following most tutorials/videos, the boxes are on the main model quickpick selection. Either way, that'd be super low on the list of things I'm sure. :)

Thanks again guys, I really appreciate it! Happy for this to be closed. Like I said, I'm sure it was just me doing PEBCAK. lol
